docs/guides/admin/docs/modules/paella.player7/plugins/org.opencast.paella.downloadsPlugin.md

```diff
@@ -0,0 +1,60 @@
+Paella plugin: org.opencast.paella.downloadsPlugin
+=======================================================
+
+This plugin adds a panel to download the videos.
+
+The configurations for this plugin are done for each tenant. So you need to modify the `plugins`
+section of the [paella config file](../configuration.md).
+
+
+Configuration
+-------------
+
+You need to enabled the `org.opencast.paella.downloadsPlugin` plugin.
+
+```json
+{
+    "org.opencast.paella.downloadsPlugin": {
+        "enabled": true,
+
+        "downloadFlavors": false,
+        "downloadTags": false,
+        "downloadMimeTypes": ["audio/m4a", "video/mp4"],
+
+        "enableOnlyLicenses": [
+            "CC-BY",
+            "CC-BY-SA",
+            "CC-BY-ND",
+            "CC-BY-NC",
+            "CC-BY-NC-SA",
+            "CC-BY-NC-ND",
+            "CC0"
+        ],
+        "enableOnWritePermission": true
+    }    
+}
+```
+
+Configuration parameters:
+
+- **enableOnlyLicenses**: The plugin is enabled if the `episode license` match any of the the licenses
+    
+    Valid values: `false` / Array strings
+
+    Example: `[ "CC-BY", "CC-BY-SA" ]`
+
+- **enableOnWritePermission**: The plugin is enabled if the `user` has `write` permission on the episode.
```

Comment:
So this supersedes the license check above, correct?  That should probably be documented as well.  I can't see this being a commonly encountered usecase but I'm sure someone's going to complain otherwise.

Otherwise, looks good to me!

Reply:
No, it's more like an `or`. The `enableOnlyLicenses` and `enableOnWritePermission` properties control when the plugin is activated. If any of them are valid, the plugin is shown.

For example. In our case we have the plugin activated as:
```json
{
"enableOnlyLicenses": ["CC-BY", ...]
"enableOnWritePermission": true
}
```
With this we achieve that if the video has a public license, the plugin will be displayed. If the video has a private license, the plugin will show only if you have `write` permissions (you are the author of the video or someone with permissions on the video, but it is not shown to students).

Reply:
I have renamed the property `enableOnlyLicenses` to `enableOnLicenses` to avoid confusion and have added an explanation to the documentation.